job_export.py

```diff
@@ -159,13 +161,57 @@
         datetime.now().strftime("export_%Y-%m-%dT%H:%M:%S")
     )
     # print(f'debug - avro_filename: {avro_filename}')
-    s3file = s3upload_file(
-        pelican_creds["manifest_bucket_name"],
-        avro_filename,
-        pelican_creds["aws_access_key_id"],
-        pelican_creds["aws_secret_access_key"],
-        fname,
-    )
+
+    # check output file format
+    if output_file_format.upper() == "ZIP":
+        print("debug - output file format is ZIP")
+        print(fname)
+        print(f"{os.path.dirname(pfb_to_zip.__file__)}" + "/config.py")
+        # Create the new file with the name specified by avro_filename
+        output_dir = os.path.dirname(fname)
+        avro_path = os.path.join(output_dir, avro_filename)
+        with open(fname, "rb") as source_file, open(avro_path, "wb") as target_file:
+            # Read the contents of fname and write them to avro_filename
+            target_file.write(source_file.read())
```

Comment:
Could you explain this to me? We need to persist to filesystem while before was just persisted directly to S3?

Reply:
We discussed this in our meeting and I believe we came to the conclusion that we need to leave it so I am marking this as resolved.